bentoml/deployment/serverless/serverless_utils.py

```diff
@@ -24,47 +24,42 @@
 from subprocess import PIPE
 
 from ruamel.yaml import YAML
-from packaging import version
 
+from bentoml.configuration import BENTOML_HOME
```

Comment:
use `from bentoml.configuration import _get_bentoml_home`, in case BENTOML_HOME got reset after loading BentoML module